server/api_conversation.go

```diff
@@ -0,0 +1,118 @@
+package main
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+	"net/http"
+
+	"github.com/mattermost/mattermost/server/public/model"
+	"github.com/mattermost/mattermost/server/public/pluginapi"
+
+	"github.com/gin-gonic/gin"
+)
+
+func (p *Plugin) handlePostConversation(c *gin.Context) {
+	userID := c.GetHeader("Mattermost-User-Id")
+
+	// We only allow bots to use this API handler for the time being.
```

Comment:
Thanks. I'll look into it although that would probably mean an additional roundtrip in my case (i.e. go through the plugin)  since the bot is just a client running outside of Mattermost.